src/Apps/gAtmoEvGen.cxx

```diff
@@ -373,15 +388,22 @@ int main(int argc, char** argv)
   // Set GHEP print level
   GHepRecord::SetPrintLevel(RunOpt::Instance()->EventRecordPrintLevel());
 
+  total_flux = atmo_flux_driver->GetTotalFluxInEnergyRange();
+  LOG("gevgen_atmo", pNOTICE) << "Total atmospheric neutrino flux is " << setprecision(2) << total_flux << " neutrinos per m^2 per second.";
+  if (gOptSecExposure > 0) {
+    /* Calculate the expected value of the total number of neutrinos we need to
+     * throw. We do this by multiplying the total flux by the exposure time in
+     * seconds and the area of the flux surface. */
+    expected_neutrinos = total_flux*gOptSecExposure*atmo_flux_driver->GetFluxSurfaceArea();
+    LOG("gevgen_atmo", pNOTICE) << "Simulating an exposure of " << setprecision(0) << gOptSecExposure << " seconds which corresponds to a total of " << setprecision(0) << expected_neutrinos << " neutrinos.";
+  }
+
   // event loop
-  for(int iev = 0; iev < gOptNev; iev++) {
+  for(int iev = 0; gOptNev > 0 ? iev < gOptNev : 1; iev++) {
```

Comment:
I suggest some ()'s here to help with readability:
```
 for (int iev = 0; ( (gOptNev > 0) ? (iev < gOptNev) : true) ; iev++)
```

Reply:
What do you think about just turning it into a while loop to make things easier to understand? I originally thought about doing that but didn't want to change too much, but now that you mention it I think it would be much easier to understand that way.